packages/table/src/common/loadableContent.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+import { Classes } from "@blueprintjs/core";
+
+export interface ILoadable {
+    /**
+     * Render the loading state.
+     * @default false
+     */
+    loading?: boolean;
+}
+
+export interface ILoadableContentProps extends ILoadable {
+    /**
+     * Show a variable length skeleton when rendering the loading state.
```

Comment:
explain what "variable length" means here (using word in its own definition). 
something about choosing a random width